flows/aws/interactive-workflow/interactive-workflow.py

```diff
@@ -56,7 +55,7 @@ def user_input_remove_features(url: str):
     raw_data = fetch(url)
 
     features = "\n".join(raw_data.get("results")[0].keys())
-
+    print(f"type(features): {type(features)}")
```

Comment:
Is this needed?

Reply:
Looks like one of my debug statements snuck in, I'll get rid of that. Thank you for catching!